api/src/main/java/io/opencensus/metrics/export/Metric.java

```diff
@@ -103,37 +103,59 @@ private static Metric createInternal(
   public abstract List<TimeSeries> getTimeSeriesList();
 
   private static void checkTypeMatch(MetricDescriptor.Type type, List<TimeSeries> timeSeriesList) {
+    
+    // Iterate though the object timeSeriesList aslong as its != null
     for (TimeSeries timeSeries : timeSeriesList) {
       System.out.println(Thread.currentThread().getStackTrace()[1].getMethodName() + 1);
+      
+      // Iterate through timeSeries.getPoints aslong as its != null
       for (Point point : timeSeries.getPoints()) {
         System.out.println(Thread.currentThread().getStackTrace()[1].getMethodName() + 2);
         Value value = point.getValue();
         String valueClassName = "";
+        
+        // enter branch if the superclass for value != null
         if (value.getClass().getSuperclass() != null) { // work around nullness check
           System.out.println(Thread.currentThread().getStackTrace()[1].getMethodName() + 3);
           // AutoValue classes should always have a super class.
           valueClassName = value.getClass().getSuperclass().getSimpleName();
         }
+        
+        // enter switch block
         switch (type) {
+
+          // case without purpose
```

Comment:
Not without purpose, this is a fallthrough (keeps going until the break)

Reply:
Also later

Reply:
Hmm... but how does it have a purpose? If it enters the case, no statements gets executed, and same thing happens if the case never gets entered.